appveyor.yml

```diff
@@ -24,12 +24,13 @@ environment:
   NOT_CRAN: true
   KEEP_VIGNETTES: false
   R_BUILD_ARGS: --no-build-vignettes --no-manual
-  R_CHECK_ARGS: --no-multiarch --no-manual --as-cran
+  R_CHECK_ARGS: --no-multiarch --no-manual --as-cran --no-build-vignettes
```

Comment:
Here vignettes check does not make any sense anyway, because vignettes are not even build.
It just produces false alarms in the build log.